### PR TITLE
[2pt] Fix Molecule tests, CI and Shell scripts

### PR DESCRIPTION
**Pull request by hackallcode**
- Fix Shell scripts to run on OS X
- Update Python dependencies
- Fix tests and CI scripts seeing new Molecule

### CODE REVIEW

**Comment by hackallcode** [comment]
The CI step "Create Test Packages" and, accordingly, the `create-packages.sh` script were tested on the first run of current commit tests. On the second run of the tests, the CI step "Cache test packages" was tested.

File: module_utils/helpers.py [hunk 1]
```diff
@@ -61,12 +61,12 @@ def __init__(self):
...
     'feedback_interval',
-])
+}
 
-memory_size_box_cfg_params = set([
+memory_size_box_cfg_params = {
     'memtx_memory',
     'vinyl_memory',
-])
+}
```
Comment (dokshina) [inline module_utils/helpers.py:69]: Please, explain this change.

Comment (hackallcode) [reply module_utils/helpers.py:69]: `set(['a', 'b', 'c'])` first creates a list, and then converts it to a set. `{'a', 'b', 'c'}` immediately creates a set. The IDE also generates a warning when you use the `set(['a', 'b', 'c'])` syntax.

But if you think that the previous version was more readable, then I will return it.

Comment (dokshina) [reply module_utils/helpers.py:69]: I'm not sure that this change should be performed in this patch. You can perform it in separated commit ant merge this PR without squash.

Comment (hackallcode) [reply module_utils/helpers.py:69]: Okey, I made two commits

**Review by dokshina — APPROVED** [review]
Thanks a lot!

**Pull request merged**